
public class 异常 {
	public static void main(String[] args) {
	/*
--------------------------------------------------------------------------------------
捕获异常
	 使用 try 和 catch 关键字可以捕获异常。try/catch 代码块放在异常可能发生的地方。 
	 
	 	try
		{
   			// 程序代码
			}catch(ExceptionName e1)
		{
   			//Catch 块
		}
--------------------------------------------------------------------------------------		
多重捕获块
	一个 try 代码块后面跟随多个 catch 代码块的情况就叫多重捕获。
	多重捕获块的语法如下所示：
	try{
   		// 程序代码
	}catch(异常类型1 异常的变量名1){
  		// 程序代码
	}catch(异常类型2 异常的变量名2){
  		// 程序代码
	}catch(异常类型3 异常的变量名3){
  		// 程序代码
	}
--------------------------------------------------------------------------------------
finally关键字
	finally 关键字用来创建在 try 代码块后面执行的代码块。
	无论是否发生异常，finally 代码块中的代码总会被执行。 
		
		 throws/throw 关键字：
		 如果一个方法没有捕获到一个检查性异常，那么该方法必须使用 throws 关键字来声明。throws 关键字放在方法签名的尾部。
		 也可以使用 throw 关键字抛出一个异常，无论它是新实例化的还是刚捕获到的。
		 下面方法的声明抛出一个 RemoteException 异常：
		 
		 import java.io.*;
		 public class className
		 {
		   public void deposit(double amount) throws RemoteException
		   {
		     // Method implementation
		     throw new RemoteException();
		   }
		   //Remainder of class definition
		 }
--------------------------------------------------------------------------------------	
	 */

	}
}
